packages/carbon-addons-devenv/sample_renderers/logo/LogoRenderer.js

```diff
@@ -4,7 +4,7 @@
 // NOTE:
 // If you want to use Renderer in the custom-carbon-addons package use the following import:
 // import { Renderer } from 'devenv_pkg/src/apollo-client-hooks';
-// eslint-disable-next-line import/no-unresolved
+// eslint-disable-next-line importPlugin/no-unresolved
```

Comment:
`import` is a reserved word, so the package now gets imported as [`importPlugin`](https://github.com/merative/spm-ui-addon-devenv/blob/485d8ec340d720951568385e0e0916e4c487e8f9/eslint.config.mjs#L4), hence we have to change the references everywhere.